### DISCUSSION TITLE
Planning Poker Ides Notes

### DISCUSSION THREAD

**Discussion by JeongJaeSoon** [2P3S/scrum-dice#13] [Ideas]
# Planning Poker 에 사용될 아이디어를 정리

**Comment by hyunnnn98** [comment]
- **크롬 익스텐션 지원**
  - 룸 생성 단축키
  - 공유버튼
  - 자동으로 클립보드에 저장
- **플래닝포커 서비스**
  - 자동으로 평균 계산
  - 가장 높은 사람과 낮은 사람이 주목되게 보이는 ( 포인트 평균 산출에 필요한 디스커션을 자연스럽게 리드해줄 수 있을듯 )
  - 매회차 기록 저장해서 보여주기
  - 쿠키에 sp 값 커스텀 가능하게하기 (0.5,1,3,4,5,7,9...)
  - 전체적인 디자인 느낌은 주사위 처럼
